stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py

```diff
@@ -39,6 +39,7 @@ def create_collection(
     ) -> schemas.Collection:
         """Create collection."""
         data = self.collection_table.from_schema(model)
+        data.type = "collection"
```

Comment:
This is most certainly not going to be kept. It ought to be removed before the PR is accepted as it is residue from debugging